### PR TITLE
Update URL of "DirectCurrent Motor Module"

### PR DESCRIPTION
**Pull request by adrian2793**
Name Request is still in progress.

I hope that's not a problem but I want to be sure you know that.

### CODE REVIEW

**Comment by adrian2793** [comment]
@per1234 I changed the url again (last time i do this, i promise).. what now?

the new url is `https://github.com/arduino279/DirectCurrent-Motor-Module`

**Comment by per1234** [comment]
> what now?

Same as before. I have confirmed your request. Now the only thing remaining is for the Library Manager index maintainer to update the database accordingly.

They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/12/06 16:46:30 Loaded 6429 libraries from DB
Removing Engine Control
Success!
```

**Pull request merged**
